dotnet/src/Connectors/Connectors.AI.OpenAI/ChatCompletion/AzureOpenAIChatCompletionService.cs

```diff
@@ -19,6 +19,21 @@ namespace Microsoft.SemanticKernel.Connectors.AI.OpenAI.ChatCompletion;
 /// </summary>
 public sealed class AzureOpenAIChatCompletionService : IChatCompletionService, ITextGenerationService
 {
+    #region BreakingGlass Properties
+    /// <summary>
+    /// Model Id for OpenAI or Deployment Name when using Azure OpenAI.
+    /// </summary>
+    public string DeploymentOrModelName => this._core.DeploymentOrModelName;
```

Comment:
This is not breaking glass as it's set by the developer.

These values are available via `IAIService.Attributes` already so no need to add these here.

Also we should not be exposing `DeploymentorModelName` as a property. We have separate properties for DeploymentName and ModelId.

Reply:
Removed.